adoc/bp_chap_getting_started_with_salt_formulas.adoc

```diff
@@ -23,15 +23,17 @@ The following list will help you in making the decision to use a state or a form
 * For large, non-trivial configurations use formulas.
 * Formulas and States both act as a kind of configuration documentation.
 Once written and stored you will have a snapshot of what your infrastructure should look like.
-* You can greatly decrease the amount of time required for writing a formula by grabbing one of the many pre-written formulas on github.
-You can find these located at https://github.com/saltstack-formulas.
+* You can decrease the amount of time required for writing a formula by grabbing a pre-written one from the https://github.com/saltstack-formulas[Saltstack formula repository on Github].
```

Comment:
Can we replace 'grabbing' with something less violent? Actually, the whole 'decrease the amount of time' bit is probably redundant. Try this re-write:

`Pre-written formulas are available from the https://github.com/saltstack-formulas[Saltstack formula repository on Github].
Use these as a starting point for your own custom formulas.`